tests/unit/test_textUtils.py

```diff
@@ -364,81 +364,81 @@ def test_normalizedOffsetsMixedIJ(self):
 class TestUniscribeSplitAtCharacterBoundaries(unittest.TestCase):
 	"""Several tests for the splitAtCharacterBoundaries function."""
 
-	def _testHelper(self, input: str, expected: list[str] | None = None) -> None:
-		if expected is None:
-			expected = list(input)
+	def _testHelper(self, input: str, expected: list[str]) -> None:
 		self.assertSequenceEqual(list(splitAtCharacterBoundaries(input)), expected)
 
 	def test_emptyString(self):
-		self._testHelper("")
+		self._testHelper("", [])
 
 	def test_singleBasicCharacter(self):
-		self._testHelper("a")
+		self._testHelper("a", ["a"])
 
 	def test_multipleBasicCharacters(self):
-		self._testHelper("Hello")
+		text = "Hello"
+		self._testHelper(text, list(text))
 
 	def test_longSentence(self):
-		self._testHelper("This is a longer sentence, with punctuation!")
+		text = "This is a longer sentence, with punctuation!"
+		self._testHelper(text, list(text))
 
 	def test_emojis(self):
-		self._testHelper("😊🤦👍")
+		text = "😊🤦👍"
+		self._testHelper(text, list(text))
 
 	def test_compositeCharacters(self):
 		self._testHelper("áéĳ", ["á", "é", "ĳ"])
 
 	def test_singleAcute(self):
-		self._testHelper("\u0301")
+		self._testHelper("\u0301", ["\u0301"])
 
-	def test_AcuteWithSpaceBefore(self):
-		# The accute is bound to the space
+	def test_acuteWithSpaceBefore(self):
+		# The acute is bound to the space
 		self._testHelper(" \u0301", [" \u0301"])
 
-	def test_accuteWithSpaceAfter(self):
-		self._testHelper("\u0301 ")
+	def test_acuteWithSpaceAfter(self):
+		self._testHelper("\u0301 ", ["\u0301", " "])
 
 	def test_sentenceWithComposites(self):
-		self._testHelper(
-			"Één eigenwĳze geïnteresseerde ĳsbeer",
-			[
-				"É",
-				"é",
-				"n",
-				" ",
-				"e",
-				"i",
-				"g",
-				"e",
-				"n",
-				"w",
-				"ĳ",
-				"z",
-				"e",
-				" ",
-				"g",
-				"e",
-				"ï",
-				"n",
-				"t",
-				"e",
-				"r",
-				"e",
-				"s",
-				"s",
-				"e",
-				"e",
-				"r",
-				"d",
-				"e",
-				" ",
-				"ĳ",
-				"s",
-				"b",
-				"e",
-				"e",
-				"r",
-			],
-		)
+		text = "Één eigenwĳze geïnteresseerde ĳsbeer"
+		expected = [
+			"É",
+			"é",
+			"n",
+			" ",
+			"e",
+			"i",
+			"g",
+			"e",
+			"n",
+			"w",
+			"ĳ",
+			"z",
+			"e",
+			" ",
+			"g",
+			"e",
+			"ï",
+			"n",
+			"t",
+			"e",
+			"r",
+			"e",
+			"s",
+			"s",
+			"e",
+			"e",
+			"r",
+			"d",
+			"e",
+			" ",
+			"ĳ",
+			"s",
+			"b",
+			"e",
+			"e",
+			"r",
+		]
+		self._testHelper(text, expected)
 
 	def test_hebrew(self):
 		self._testHelper("בְּרֵאשִׁית", ["בְּ", "רֵ", "א", "שִׁ", "י", "ת"])
```

Comment:
I'm extremely confused with this syntax and how it's passing. Shouldn't it be?

```suggestion
		self._testHelper(["בְּרֵאשִׁית", ["בְּ", "רֵ", "א", "שִׁ", "י", "ת")
```

Reply:
No. The first parameter is what you pass to `textUtils.uniscribe.splitAtCharacterBoundaries`, namely a string. The second parameter is what you expect from it. `textUtils.uniscribe.splitAtCharacterBoundaries` returns an iterator, which we convert to a list in the testHelper.